js/components/pages/PassagePage.react.js

```diff
@@ -20,6 +21,9 @@ class PassagePage extends Component {
       previousButton = <button className="previous" onClick={() => { dispatch(asyncPreviousVerse()) }}>Previous</button>
     }
 
+    const audioUrl = "http://www.esvapi.org/v2/rest/passageQuery?key=IP&passage=" +
```

Comment:
From looking at the docs we will need to replace that API key with an actual one to avoid errors for making too many queries.  I've posted the key I already got setup in Slack, will need to think about how manage that, as I would prefer not to commit it to the repo.